Automattic/jetpack
# Always use house icon for site name in admin-bar

Fixes https://github.com/Automattic/dotcom-forge/issues/8394

## Proposed changes:

* This PR overrides the "speedometer" icon in the admin-bar to the house icon when the user is on the site's front page.

Before | After
--|--
<img width="626" alt="Screenshot 2024-07-22 at 11 22 13 AM" src="https://github.com/user-attachments/assets/aba917ee-7c5b-47c5-a337-d4c5989f94a7"> | <img width="623" alt="Screenshot 2024-07-22 at 11 22 20 AM" src="https://github.com/user-attachments/assets/cd4761d6-4797-4a0b-af0e-3998c876ffe1"> 

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?


No.

## Testing instructions:






* Load this PR on an Atomic site and a Simple site using the [instructions below](https://github.com/Automattic/jetpack/pull/38457#issuecomment-2243345660).
* Go to your test site's front page and see that the house icon replaces the "speedometer" icon.
* Click it to make sure it works.
* Test it on mobile and desktop.
